LICENSE.txt

```diff
@@ -1,5 +1,5 @@
 BSD 3-clause license
-Copyright (c) 2015-2019, conda-forge
+Copyright (c) 2015-2020, conda-forge contributors
```

Comment:
What is being copyrighted here? Typeguard uses the MIT license, shouldn't that be reflected here?

Reply:
You should be able to fix that (and merge this PR) but I can also deal with it when I'm back from vacation later this week.